flask_app/controllers/projects.py

```diff
@@ -59,4 +62,4 @@ def edit_project (id):
     data = {
         "id" : id,  
     }
-    return render_template("update.html", user=User.get_by_id(data), "Place Holder" = Project.get_one_project(data))
+    return render_template("update.html", user=User.get_by_id(data), project_to_edit = Project.get_one_project(data))
```

Comment:
just changing this since it was a placeholder

Reply:
Perfect! Please make changes as you see fit that will help assist you on the front-end